mandatory/1-fix-functions.js

```diff
@@ -10,7 +10,7 @@
 
 */
 
-function mood() {
+function mood(isHappy) {
   let isHappy = true;
```

Comment:
Kindly note, when you say "let isHappy = true", you are changing the passed in the argument of the mood to "true" all the time.

Meaning you will always get true as an answer.

Have you tried removing this line?